lib/active_model/serializer/attributes.rb

```diff
@@ -14,10 +14,12 @@ module Attributes
 
         # Return the +attributes+ of +object+ as presented
         # by the serializer.
-        def attributes(requested_attrs = nil, reload = false)
+        def attributes(options = {}, reload = false)
+          requested_attrs = options[:only]
+          excepts = Array(options[:except])
```

Comment:
I'm not sure I follow; if/unless seems like it can prevent the entire association from being included/rendered, where `except` will still render the association but exclude some specific fields.